apps/auth0/src/post-login-connect-user.ts

```diff
@@ -2,6 +2,27 @@ import { Auth0PostLoginApi } from '@vedicium/auth0-actions-sdk';
 import got from 'got';
 import type { Auth0PostLoginEventWithSecrets } from './types';
 
+const getApiUrl = (event: Auth0PostLoginEventWithSecrets) => {
```

Comment:
any chance this could be made reusable across both actions? the input event is a little different but we only use the `event.request` which I think should be the same anyway?

Reply:
No. Need rollup for that